hw/dv/data/xcelium/xcelium.hjson

```diff
@@ -11,6 +11,8 @@
 
   build_opts: ["-elaborate -64bit -access +r -sv",
                "-licqueue",
+               // allow same module name as vendor lib (like ibex) has some same modules as OT
```

Comment:
Please turn this into TODO and attach #1231 to this. Once that is resolved, we should remove this option.

Reply:
Thanks, updated.